Resources/Locale/en-US/headset/headset-component.ftl

```diff
@@ -7,3 +7,14 @@ examine-headset = A small screen on the headset displays the following available
 examine-headset-channel = [color={$color}]:{$key} for {$id} ({$freq})[/color]
 
 examine-headset-chat-prefix = Use {$prefix} for the currently tuned frequency.
+
+chat-radio-common = Common
+chat-radio-centcom = CentCom
+chat-radio-command = Command
+chat-radio-engi = Engineering
+chat-radio-med = Medical
+chat-radio-sci = Science
+chat-radio-sec = Security
+chat-radio-serv = Service
+chat-radio-sup = Supply
+chat-radio-syndie = Syndicate
```

Comment:
Don't abbreviate localization names. i.e. `chat-radio-serv` should be `chat-radio-service`